tests/test_task_build.py

```diff
@@ -32,3 +33,19 @@ def test_mkdir(tmpdir):
 
     mkdir(test_file)
     assert os.path.isfile(test_file)
+
+
+def test_run_cmd(tmpdir):
+    """Tests for run_cmd function."""
+    output, exit_code, exit_error = run_cmd("echo hello", 'test', tmpdir)
+    assert output == "hello\n"
+    assert exit_code == 0
+    assert exit_error == ""
+
+    output, exit_code, exit_error = run_cmd("ls -l /does_not_exists.txt", 'fail test', tmpdir)
+    assert exit_code != 0
+    assert exit_error != ""
+ 
+    output, exit_code, exit_error = run_cmd("this_command_does_not_exist", 'fail test', tmpdir)
+    assert exit_code != 0
+    assert exit_error != ""
```

Comment:
Next to fixing the trivial code style issues reported by the Hound bot (and `flake8`), we should also:
* take into account the changed order of return values for `run_cmd`
* always check exit code + both stdout/stderr in each test case
* have a more specific test and just `!= ""` for the stderr output

Like this:

```python
def test_run_cmd(tmpdir):
    """Tests for run_cmd function."""
    out, err, ec = run_cmd("echo hello", 'test', tmpdir)
    assert ec == 0
    assert out == "hello\n"
    assert err == ""

    out, err, ec = run_cmd("ls -l /does_not_exists.txt", 'fail test', tmpdir)
    assert ec != 0
    assert out == ""
    assert "No such file or directory" in err

    out, err, ec = run_cmd("this_command_does_not_exist", 'fail test', tmpdir)
    assert ec != 0
    assert out == ""
    assert "this_command_does_not_exist: command not found" in err